charpick/charpick.c

```diff
@@ -350,478 +362,540 @@ chooser_button_clicked (GtkButton *button, charpick_data *curr_data)
+                            TRUE, TRUE, 0);
+    }
+
+    g_free (toggle_button);
+    g_free (row_box);
+
+    gtk_container_add (GTK_CONTAINER (p_curr_data->applet),
+                       box);
+    gtk_widget_show_all (p_curr_data->box);
+
+    p_curr_data->last_index = NO_LAST_INDEX;
+    p_curr_data->last_toggle_button = NULL;
 }
 
-static void applet_size_allocate(MatePanelApplet *applet, GtkAllocation *allocation, gpointer data)
+static void
+applet_size_allocate (MatePanelApplet *applet,
+                      GtkAllocation   *allocation,
+                      gpointer         data)
 {
-  charpick_data *curr_data = data;
-  if (curr_data->panel_vertical) {
-    if (curr_data->panel_size == allocation->width)
-      return;
-    curr_data->panel_size = allocation->width;
-  } else {
-    if (curr_data->panel_size == allocation->height)
-      return;
-    curr_data->panel_size = allocation->height;
-  }
-
-  build_table (curr_data);
-  return;
+    charpick_data *curr_data = data;
+    if (curr_data->panel_vertical) {
+        if (curr_data->panel_size == allocation->width)
+            return;
+        curr_data->panel_size = allocation->width;
+    } else {
+        if (curr_data->panel_size == allocation->height)
+            return;
+        curr_data->panel_size = allocation->height;
+    }
+
+    build_table (curr_data);
+    return;
 }
 
-static void applet_change_orient(MatePanelApplet *applet, MatePanelAppletOrient o, gpointer data)
+static void
+applet_change_orient (MatePanelApplet      *applet,
+                      MatePanelAppletOrient o,
+                      gpointer              data)
 {
-  charpick_data *curr_data = data;
-  if (o == MATE_PANEL_APPLET_ORIENT_UP ||
-      o == MATE_PANEL_APPLET_ORIENT_DOWN)
-    curr_data->panel_vertical = FALSE;
-  else
-    curr_data->panel_vertical = TRUE;
-  build_table (curr_data);
-  return;
+    charpick_data *curr_data = data;
+    if (o == MATE_PANEL_APPLET_ORIENT_UP ||
+        o == MATE_PANEL_APPLET_ORIENT_DOWN)
+        curr_data->panel_vertical = FALSE;
+    else
+        curr_data->panel_vertical = TRUE;
+
+    build_table (curr_data);
+    return;
 }
 
-
 static void
 about (GtkAction     *action,
        charpick_data *curr_data)
 {
-  static const char * const authors[] = {
-	  "Alexandre Muñiz <munizao@xprt.net>",
-	  "Kevin Vandersloot",
-	  NULL
-  };
-
-  static const gchar* documenters[] = {
-          "Dan Mueth <d-mueth@uchicago.edu>",
-          N_("Sun GNOME Documentation Team <gdocteam@sun.com>"),
-          N_("MATE Documentation Team"),
-	  NULL
-  };
+    static const char * const authors[] = {
+            "Alexandre Muñiz <munizao@xprt.net>",
+            "Kevin Vandersloot",
+            NULL};
+
+    static const gchar* documenters[] = {
+            "Dan Mueth <d-mueth@uchicago.edu>",
+            N_("Sun GNOME Documentation Team <gdocteam@sun.com>"),
+            N_("MATE Documentation Team"),
+            NULL};
```

Comment:
When you declare and initialize a variable the closing bracket is aligned with the 1st line.

Reply:
Done.